server/sched.go

```diff
@@ -723,6 +727,25 @@ func pickBestFitGPUs(req *LlmRequest, ggml *llm.GGML, gpus gpu.GpuInfoList, numP
 	return nil
 }
 
+// If multiple Libraries are detected, pick the Library which loads the most layers for the model
+func pickBestPartialByLibrary(req *LlmRequest, ggml *llm.GGML, gpus gpu.GpuInfoList, numParallel *int) gpu.GpuInfoList {
```

Comment:
nit for later: we should find a way to refactor this to not mutate `numParallel` along the way (vs returning it somewhere) – it can become difficult to track its value through the scheduler logic

Reply:
I'll file an issue so this doesn't slip through the cracks.